### ISSUE TITLE
Asset tag not working

### ISSUE THREAD

**Issue by jonassiewertsen** [statamic/cms#2100]
The asset tag is simply returning nothing.

[Tag documentation](https://statamic.dev/tags/asset)

This might be related to https://github.com/statamic/cms/issues/1155

## Environment

**Statamic version:** 3.0.0.-beta-37

**PHP version:** 7.4

**Install method (choose one):**
- Fresh install from `statamic/statamic`

**Comment by caseydwyer** [comment]
We're seeing this issue as well. Using the S3 driver for this particular asset container (in case that's helpful/relevant).

**Comment by jackmcdade** [comment]
@jonassiewertsen It's working for me. Is the file you're retrieving inside an asset container? It *must* be for it to work.

**Comment by jonassiewertsen** [comment]
@jackmcdade It's in the default assets container. So that should be alright. 

Yesterday I did some tinkering with the Asset Facade. By default I couldn't get a return. 
If passing `containername::filename` I did receive the asset. That's still buggy, because I couldn't fetch automatically in which container the asset has been. 

I would guess it's connected to the selection of the default container. There has been an issue a few months ago (which should be resolved).
If only one container is presented, this container should be set as default container. Maybe that is not happening, so the Asset tag does know where to look? Just guessing obviously ...
